task/git-clone/0.2/git-clone.yaml

```diff
@@ -68,14 +68,18 @@ spec:
       description: log the commands used during execution
       type: string
       default: "true"
+    - name: gitInitImage
```

Comment:
We will need to add this param to the README of the task as well. 
Can you please add that [here](https://github.com/tektoncd/catalog/blob/master/task/git-batch-merge/0.2/README.md)

Reply:
And Probably in other PR we will have to modify/cleanup README of this task like we have for other git [tasks](https://github.com/tektoncd/catalog/tree/master/task/git-cli/0.1)

Reply:
Sure 🙂